product_analytic_purchase/__manifest__.py

```diff
@@ -1,11 +1,11 @@
 # Copyright 2017 Akretion (http://www.akretion.com/) - Alexis de Lattre
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
 {
     "name": "Product Analytic Purchase",
-    "version": "10.0.1.0.0",
+    "version": "14.0.1.0.0",
     "category": "Purchases",
-    "license": "AGPL-3",
+    "license": "LGPL-3",
```

Comment:
@Reyes4711 Is there a reason to change the license here ?

Reply:
@rousseldenis I made an error with this, sorry. I restore this changes. Can you approve this?